src/TuDa.CIMS.Web/Components/WorkingGroupList/WorkingGroupPageWorkingGroupList.razor.cs

```diff
@@ -57,6 +57,11 @@ IEnumerable<WorkingGroup> workingGroups
             );
     }
 
+    public async Task ReloadDataGridAsync()
+    {
+        await _dataGrid.ReloadServerData();
+    }
```

Comment:
⛏️ This could be simplified to this
```suggestion
    public Task ReloadDataGridAsync() => _dataGrid.ReloadServerData();
    
```